test/warning_test.py

```diff
@@ -4,7 +4,7 @@
 >>> import json
 >>> from samila import *
 >>> from pytest import warns
->>> with warns(RuntimeWarning, match="Neither function nor data is provided."):
+>>> with warns(RuntimeWarning, match="Neither function nor data is provided. Samila will switch to random equations mode."):
```

Comment:
I think we don't need to raise warning anymore. Till now not providing data or function meant to be a warning but now the only way by which user can use the random mode is not providing them (and maybe the most usable feature of Samila) so I think we don't need this warning anymore.